maker/src/lib.rs

```diff
@@ -38,6 +38,11 @@ pub struct Opts {
     #[clap(short, long)]
     pub json: bool,
 
+    /// If enabled the application will fail fast and terminate if an error occurred during db
+    /// migration.
+    #[clap(short, long)]
+    pub db_failsafe: bool,
```

Comment:
🔧 I think we should default to a value of `true` so that we can always safely omit this argument.

Reply:
Fair. What about the name `ignore-migration-errors`? It sounds pretty natural.

Reply:
can do :)